docs/scenarios/library-authoring/pinning-libraries-to-rc2.md

```diff
@@ -0,0 +1,70 @@
+
+**Trimming** - The act of removing the packages you do not depend on from a metapackage.  The end result is that your library only depends on packages it uses.  See guidance in the libraries doc for more information.
+
+## Fasten your dependencies to RC2
+
+When creating libraries with .NET Core RC2, it's important that you fasten your dependencies to RC2.  This means two things.  First, you should only depend on packages marked as final for .NET Core RC2.  Second, each package should come from the same build. This means every package should have this pattern: `<package_version>-rc2-<build_number>`.
+
+**Note: The build number at the end is not final right now.  It will have a final number when RC2 ships.**
+
+**Examples of packages fastened to RC2**
+
+`"System.Collections":"4.0.11-rc2-24022"`
+
+`"NETStandard.Library":"1.5.0-rc2-24022"`
+
+`"Microsoft.NETCore.App":"1.0.0-rc2-24022"`
+
+**Examples of packages that are NOT fastened to RC2**
+
+`"System.Collections":"4.0.11-rc3-24021-00"`
+
+`"System.Net.Http":"4.1.0-rc2-24008"`
+
+`"System.Text.RegularExpressions":"4.0.10-rc3-24021-00"`
+
+### Why does this matter?
+
+We guarantee that if you fastened your dependencies to RC2, those packages will all work with together.  There is no such guarantee if you use packages which aren't fastened to RC2.
+
+### Is there some big list to look at with version numbers and stuff?
+
+Yes, but depending on the nature of your library, you may not need to look through a big list.
+
+**Are you depending only on `NETStandard.Library`?**
+
+If so, all you need to do is fastened your `NETStandard.Library` package to RC2.  Because this is a curated metapackage, its package closure is also fastened to RC2.
+
+**Are you depending only on `Microsoft.NETCore.App`?**
+
+If so, all you need to do is fastened your `Microsoft.NETCore.App` package to RC2.  Because this is a curated metapackage, its package closure is also fastened to RC2.
+
+**Are you trimming your `NETStandard.Library` or `Microsoft.NETCore.App` metapackage dependencies?**
+
+If so, all you need to do is ensure that the metapackage you start with is fastened to RC2.  The individual packages you depend on after trimming are also fastened to RC2.
+
+**Are you depending on packages outside the NETStandard.Library or Microsoft.NETCore.App metapackages?**
+
+If so, you need to fastened your other dependencies to RC2.  Read below.
+
+### A note on using a splat string (\*) when versioning
+
+You may have adopted a versioning pattern which uses a splat (\*) string like this:
+`"System.Collections":"4.0.11.-rc2-*"`.
+
+**You should not do this**.  Pick a specific number and use that number for all your package dependencies.  If you do so, you could end up pulling in packages from different builds.  This could then result in some packages being incompatible.
```

Comment:
The second sentence here seems out of place. "do so" I think implies using the 'splat string', but the other sentence describe *not* using the 'splat string'.